sdk/servicebus/azure-messaging-servicebus/src/main/java/com/azure/messaging/servicebus/ServiceBusReceiverAsyncClient.java

```diff
@@ -36,9 +37,25 @@
  * An <b>asynchronous</b> receiver responsible for receiving {@link ServiceBusReceivedMessage} from a specific queue or
  * topic on Azure Service Bus.
  *
+ * <p><strong>Create an instance of receiver</strong></p>
+ * {@codesnippet com.azure.messaging.servicebus.servicebusasyncreceiverclient.instantiation}
+ *
+ * <p><strong>Create an instance of sender using default credential</strong></p>
+ * {@codesnippet com.azure.messaging.servicebus.servicebusasyncreceiverclient.instantiateWithDefaultCredential}
+ *
+ * <p><strong>Create a sender and receive message from Service Bus resource</strong></p>
+ * {@codesnippet com.azure.messaging.servicebus.servicebusasyncreceiverclient.receive#message}
+ *
+ * <p><strong>Create a sender and receive all the messages from Service Bus resource</strong></p>
+ * {@codesnippet com.azure.messaging.servicebus.servicebusasyncreceiverclient.receive#all }
```

Comment:
This is very similar to the one above. I don't think you need to show the ".take(1)" example, and instead create with some receive message options like RECEIVE_AND_DELETE or auto renew lock.